neuron/neuron/contest.py

```diff
@@ -1,11 +1,15 @@
+import shutil
 from abc import ABC, abstractmethod
 from enum import Enum
+from pathlib import Path
 
 import torch
 from diffusers import StableDiffusionXLPipeline
 
 from .coreml_pipeline import CoreMLStableDiffusionXLPipeline
 
+MODEL_CACHE_DIR = Path("models/.cache")
```

Comment:
think just `model-cache` or something would be fine, doesn't need to be a nested directory